bedrock/legal/templates/legal/impressum.html

```diff
@@ -0,0 +1,37 @@
+{% extends "base-resp.html" %}
+
+{% block page_title %}{{_('Mozilla Foundation End-User Licensing Agreements')}}{% endblock %}
+{% block body_id %}legal_eula_index-page{% endblock %}
+{% block body_class %}sand{% endblock %}
+
+{% block breadcrumbs %}{% endblock %}
+
+{% block extrahead %}
+  {% stylesheet 'legal-impressum' %}
+{% endblock %}
+
+{% block content %}
+
+<div id="main-feature">
+  <h1>Impressum</h1>
+</div>
+
+<div id="main-content">
+  <section class="intro">
+    <h2>Dienstanbieter:</h2>
+    <p>Mozilla Corporation (Gesellschaft mit beschränkter Haftung nach US-amerikanischem Recht) Corporate Number 2759208</p>
+
+    <h2>Sitz der Gesellschaften:</h2>
+    <p>313 E. Evelyn Ave., Mountain View CA 94041, USA</p>
```

Comment:
I feel that we should mark this up a bit more semantically. Something like here:
https://github.com/mozilla/bedrock/blob/master/bedrock/mozorg/templates/mozorg/contact/spaces/mountain-view.html#L11